sdk/communication/communication-administration/package.json

```diff
@@ -83,7 +83,7 @@
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@azure/test-utils-recorder": "^1.0.0",
     "@azure/identity": "^1.1.0",
-    "@microsoft/api-documenter": "~7.8.17",
+    "@microsoft/api-documenter": "^7.12.14",
```

Comment:
Why update from `~` to `^`?

Reply:
My mistake!

Reply:
It's not necessarily a mistake.  The question is why were we using `~` instead of `^` to begin with (since we use `^` by default for all GA deps).

Reply:
If we were using `~` for a good reason we should keep doing it.  But otherwise we should use `^` to align with other deps.

Reply:
I did the change without paying enough attention. I'm using ~ now